modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf

```diff
@@ -5,7 +5,7 @@ process CRAM_FILTER_MINIMAP2_FILTER5END_FIXMATE_SORT {
     container 'quay.io/sanger-tol/cramfilter_bwamem2_minimap2_samtools_perl:0.001-c1'
 
     input:
-    tuple val(meta), path(cramfile), path(cramindex), val(from), val(to), val(base), val(chunkid), val(rglines), val(ref)
+    tuple val(meta), path(cramfile), path(cramindex), val(from), val(to), val(base), val(chunkid), val(rglines), val(ref), path(reference)
```

Comment:
If these are the meta and reference. Should this not just be it's own input?

Reply:
I think val(ref) is possibly an unclear name, looking at hic_minimap2.nf it is actually the path to the minimap index.

Reply:
sorry guys, cramfilter modules take index as input not reference fasta.

Reply:
Shall I update the name then in the module?